Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM nvcr.io/nvidia/pytorch:24.01-py3
 ARG DEBIAN_FRONTEND=noninteractive
 WORKDIR /openduck-py
 
-COPY ./openduck-py /openduck-py
+# Copying is only necessary on prod, when you're not mounting the volume
+# COPY ./openduck-py /openduck-py
```

Comment:
do you know if CI deployments are working and if this will break them?

Reply:
oh not sure